translations/en.json

```diff
@@ -235,8 +235,10 @@
 		},
 		"onboard": {
 			"message": "Your wallet has no Synths. Buy <0>{{currencyKey}}</0> to trade Synths:",
-			"l2-message": "Your wallet has no Synths. Mint <0>{{currencyKey}}</0> on Staking to trade Synths:",
-			"mint-button": "Visit Staking"
+			"l2-message": "Your wallet has no Synths. Buy <0>{{currencyKey}}</0> on 1inch or Uniswap to trade Synths:",
+			"mint-button": "Visit Staking",
```

Comment:
You can probably remove`mint-button`, it's not used anywhere else within the dApp